src/polytopes/chain.jl

```diff
@@ -15,6 +15,9 @@ end
 Chain(vertices::CircularVector) =
   Chain([collect(vertices); vertices[begin]])
 
+Chain(vertices::AbstractVector) = Chain(collect(vertices))
+Chain(vertices::AbstractVector{<:Tuple}) = Chain(Point.(vertices))
+
```

Comment:
Similar for this one. Why the `Polytope` constructors are not being propagated?

Reply:
This one came from the ambiguity error above. This can be removed if we restrict the above to only handle points though, which may be better.